pipelines/template-build/template-build.yaml

```diff
@@ -223,6 +223,42 @@ spec:
         value: $(tasks.build-image-index.results.IMAGE_DIGEST)
       - name: image-url
         value: $(tasks.build-image-index.results.IMAGE_URL)
+    - name: sast-shell-check
+      when:
+        - input: $(params.skip-checks)
+          operator: in
+          values: ["false"]
+      runAfter:
+        - build-image-index
+      taskRef:
+        name: sast-shell-check
+        version: "0.1"
+      workspaces:
+        - name: workspace
+          workspace: workspace
+      params:
+        - name: image-digest
+          value: $(tasks.build-image-index.results.IMAGE_DIGEST)
+        - name: image-url
+          value: $(tasks.build-image-index.results.IMAGE_URL)
+    - name: sast-unicode-check
+      when:
+        - input: $(params.skip-checks)
+          operator: in
+          values: ["false"]
+      runAfter:
+        - build-image-index
+      taskRef:
+        name: sast-unicode-check
+        version: "0.1"
+      workspaces:
+        - name: workspace
+          workspace: workspace
+      params:
+        - name: image-digest
```

Comment:
`sast-unicode-check` (unlike `sast-shell-check`) does not have the `image-digest` param: https://github.com/konflux-ci/build-definitions/blob/c9d0d5ce3a5f8034d3f158ac649115150e694403/task/sast-unicode-check/0.1/sast-unicode-check.yaml#L16

Reply:
Fixed!